app/lib/credentialDisplay/openBadgeCredential.tsx

```diff
@@ -35,24 +36,30 @@ export const openBadgeCredentialDisplayConfig: CredentialDisplayConfig = {
 function OpenBadgeCredentialCard({ rawCredentialRecord, onPressIssuer }: CredentialCardProps): React.ReactElement {
   const { styles, theme } = useDynamicStyles(dynamicStyleSheet);
   const { credential } = rawCredentialRecord;
-  const { credentialSubject, issuer, name } = credential;
+  const { credentialSubject, issuer, name  } = credential;
 
   const issuanceDate = getIssuanceDate(credential);
   const expirationDate = getExpirationDate(credential);
   const formattedIssuanceDate = issuanceDate ? moment(issuanceDate).format(DATE_FORMAT) : 'N/A';
   const formattedExpirationDate = expirationDate ? moment(expirationDate).format(DATE_FORMAT) : 'N/A';
 
   const {
+    issuedTo,
     description,
     criteria,
-    subjectName,
     numberOfCredits,
     startDateFmt,
     endDateFmt,
     achievementImage,
     achievementType
   } = credentialSubjectRenderInfoFrom(credentialSubject);
 
```

Comment:
```suggestion
// Credley, for some reason, uses VC.name as the person's "issued to" name
```

Reply:
Line should read 'Credly', not 'Credley'.

Reply:
Whoops, fixed.